Tools/autotest/arducopter.py

```diff
@@ -1086,6 +1086,42 @@ def test_vibration_failsafe(self):
         self.context_pop()
         self.reboot_sitl()
 
+    def test_motor_failsafe_mode(self, mode, user_takeoff=False):
+        # stabilize check
+        self.progress("Motor failsafe check in %s" % mode)
+        self.change_mode(mode)
+        self.zero_throttle()
+        self.wait_ready_to_arm()
+        self.context_collect('STATUSTEXT')
+        self.send_mavlink_arm_command()
+        self.mav.recv_match(blocking=True, timeout=1)
+        if user_takeoff:
+            self.run_cmd(mavutil.mavlink.MAV_CMD_NAV_TAKEOFF, 0, 0, 0, 0, 0, 0, 10)
+        else:
+            self.set_rc(3, 1700)
+        # we may never see ourselves as armed is a heartbeat
+        self.wait_statustext("Takeoff blocked: ESC RPM too low")
+        self.zero_throttle()
+        self.disarm_vehicle()
+        self.wait_disarmed()
+
+    # Tests the motor failsafe
+    def test_motor_failsafe(self):
```

Comment:
here too, let's call it "takeoff check" or something similar.

Reply:
Fixed